### ISSUE TITLE
Keyboard shortcuts to add task

### ISSUE THREAD

**Issue by deadcoder0904** [coddx-hq/coddx-alpha#14]
When creating 2 tasks simultaneously, I want the next task to start with an empty text field after I press Enter <kbd>⏎</kbd> as Todoist does.

Or give me some keyboard shortcut like <kbd>⌘T</kbd> to add Task although I prefer the above suggestion more :)

**Comment by ngduc** [comment]
right, I'm thinking about using "Enter" (or Cmd+Enter, etc. if need to reserve for multi-line text later). also "Tab" can be used to indent to make a sub-task.

will work on this.

**Comment by mevin-e** [comment]
Indeed, this functionality will make it much faster to add new task.
